tajo-core/src/main/java/org/apache/tajo/querymaster/Repartitioner.java

```diff
@@ -740,7 +735,7 @@ public static void scheduleRangeShuffledFetches(TaskSchedulerContext schedulerCo
         fetchSet = new HashSet<FetchImpl>();
         for (FetchImpl fetch: fetches) {
           String rangeParam =
-              TupleUtil.rangeToQuery(ranges[i], ascendingFirstKey ? i == (ranges.length - 1) : i == 0, encoder);
```

Comment:
@sirpkt 
I have heard from hyunsik that descending order consider the sampling and limit clause.
In my opinion, main bug fixing  is good. 
The range array is ascending. so {{ascendingFirstKey}} checking do not need
```
TupleUtil.rangeToQuery(ranges[i], i == (ranges.length - 1), encoder);
```